requirements.txt

```diff
@@ -0,0 +1,84 @@
+aniso8601==3.0.2
```

Comment:
why do we need to add this here? I thought it already exists in the `server` directory

Reply:
@BolajiOlajide I had to upgrade a few packages and regenerate the requirements file since the `build` method of the `googleapiclient.discovery` python package was not sending calendar invites. This [link](https://github.com/googleapis/google-auth-library-python/issues/190) clearly explains the solution to what was happening while I tried to send a calendar invite. The solution is at the end of the page. I discovered what was happening during the debugging process.

Reply:
Yes, I understand that but I think this file is in the wrong directory. I believe it should be inside the `server/` directory.

Reply:
@BolajiOlajide, sorry this comes in a bit late. I just implemented your feedback a couple of minutes ago. I've updated the requirements file to show the updated packages.